contents/blog/trademarks-blog-post.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-08-18
+title: Don’t Bother Securing Your Trademarks In The Beginning
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+Don’t bother securing your trademarks in the early days of your business. 
```

Comment:
_Don’t bother securing your trademarks in the early days of your business - wait until you've raised a seed round first._

Reply:
Amended